export default async () => {
  try {
    console.log('signout')
  } catch (error) {
    console.error(error)
  }
}
